src/test/java/UploadReceiptServletTest.java

```diff
@@ -205,6 +209,8 @@ public void doPostUploadsReceiptToDatastoreDevServer() throws IOException {
     when(blobInfoFactory.loadBlobInfo(BLOB_KEY)).thenReturn(blobInfo);
 
     when(request.getParameter("label")).thenReturn(LABEL);
```

Comment:
I mentioned this in another PR but since a lot of this setup code for each test is repeated, maybe it can be put into a helper function

Reply:
I've added some helper functions here and in PR #27.